docs/reference/overview.md

```diff
@@ -31,6 +38,12 @@ Setting this is optional. If you do not set this, the `COMPOSE_PROJECT_NAME` def
 
 Specify the file containing the compose configuration. If not provided, Compose looks for a file named  `docker-compose.yml` in the current directory and then each parent directory in succession until a file by that name is found.
 
+### COMPOSE\_API\_VERSION
+
+Specify the api version the client should report. If you get an error about
+`client and server don't have same version` you may use this environment
+variable to have the client report a version compatible with the server.
```

Comment:
We had some discussion on this in docker; https://github.com/docker/docker/pull/15964#discussion_r38387671

It's not yet merged, but it will be described as a "power user" feature, and "use at your own risk"